app/views/categories/show.html.slim

```diff
@@ -8,8 +8,8 @@
         h5 #{@category.description}
       .grid-x.grid-padding-x
         .medium-6.cell
-          = link_to(@videos.last)
-            = image_tag src="http://lorempixel.com/640/480/animals"
+          = link_to(@latest_video)
```

Comment:
Add `- if @latest_video` so it does not crash when there are no videos in the category.